Symfony/CS/Fixer/Contrib/PhpUnitDedicateAssertFixer.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\ConfigurationException\InvalidFixerConfigurationException;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class PhpUnitDedicateAssertFixer extends AbstractFixer
+{
+    private $fixMap = array(
+        'array_key_exists' => array('assertArrayNotHasKey', 'assertArrayHasKey'),
+        'empty' => array('assertNotEmpty', 'assertEmpty'),
+        'file_exists' => array('assertFileNotExists', 'assertFileExists'),
+        'is_infinite' => array('assertFinite', 'assertInfinite'),
+        'is_nan' => array(false, 'assertNan'),
+        'is_null' => array('assertNotNull', 'assertNull'),
+        'is_array' => true,
+        'is_bool' => true,
+        'is_boolean' => true,
+        'is_callable' => true,
+        'is_double' => true,
+        'is_float' => true,
+        'is_int' => true,
+        'is_integer' => true,
+        'is_long' => true,
+        'is_​numeric' => true,
+        'is_object' => true,
+        'is_real' => true,
+        'is_​resource' => true,
+        'is_scalar' => true,
+        'is_string' => true,
+    );
+
+    private $configuration = array(
+        'array_key_exists',
+        'empty',
+        'file_exists',
+        'is_infinite',
+        'is_nan',
+        'is_null',
+        'is_array',
+        'is_bool',
+        'is_boolean',
+        'is_callable',
+        'is_double',
+        'is_float',
+        'is_int',
+        'is_integer',
+        'is_long',
+        'is_​numeric',
+        'is_object',
+        'is_real',
+        'is_​resource',
+        'is_scalar',
+        'is_string',
+    );
+
+    /**
+     * @param string[] $usingMethods
+     */
+    public function configure(array $usingMethods)
```

Comment:
maybe we could reconsider if the value should be nullable or not, but for now please be compatible with 2.x ;)